.npmrc

```diff
@@ -0,0 +1 @@
+@dbl-works:registry=https://npm.pkg.github.com
```

Comment:
How do we build & deploy new versions for this? Do we have e.g. a `/bin/deploy` script to automate this?

Reply:
@marfoldi

Reply:
Added in https://github.com/dbl-works/cloudflare-router/pull/21.